src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -100,6 +93,15 @@ public SearchVariantsRequest getVariantsRequest(String variantSetId) {
         .setEnd(end);
   }
 
+  public StreamVariantsRequest getStreamVariantsRequest(String variantSetId) {
```

Comment:
So instead of adding a new method here, I'd move the original method and place this one along with it.